virttest/qemu_vm.py

```diff
@@ -627,6 +627,12 @@ def add_nic(devices, vlan, model=None, mac=None, device_id=None,
                 if ctrl_mac_addr and ctrl_mac_addr in ["on", "off"]:
                     dev.set_param('ctrl_mac_addr', ctrl_mac_addr)
                 dev.set_param('mac', mac, dynamic=True)
+
+                # passing romfile= tells QEMU to disable ROM entirely for
+                # this device
+                no_romfile = params.get("net_dev_disable_option_rom", None)
```

Comment:
request to default the params used here and document about the params, its value and its use in cfg